kev1n1in/AsiaYoTravelMap
# feature/streetViewModal

## Description

Adds a new Street View modal to the project and implements a toggle functionality that allows users to switch between the existing image modal and the new Street View modal.

## Test Plan

- Open a modal in the map.
- Verify that the modal initially displays the image mode.
- Click the "切換到街景模式" button and verify that the modal content switches to the Street View mode.
- Click the "切換到圖片模式" button and verify that the modal content switches back to the image mode.
- Test the toggle functionality across different places to ensure it works consistently.
- Verify that the layout and elements within the modal (such as buttons, date picker, and rating) are displayed correctly in both modes.
